distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java

```diff
@@ -484,7 +484,8 @@ private static Integer findPortNumber(Integer portToTry) {
         // increase timeout for CI environment
         systemTimeout(GENERIC_TIMEOUT_MILLISECONDS),
         when(Boolean.getBoolean("keepRuntimeFolder")).useOptions(keepRuntimeFolder()),
-        cleanCaches(true));
+        cleanCaches(true),
+        KarafDistributionOption.debugConfiguration());
```

Comment:
❗️ This is a duplicate of https://github.com/codice/ddf/blob/master/distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/AbstractIntegrationTest.java#L565 . It also hardcodes the debug port to 5005 which we should make dynamic.

If we executed the debugger through pax exam, we could take advantage of the `hold` flag parameter which will wait for the the debugger to attach before starting the test.